Solutions/Tomcat/Hunting Queries/TomcatTopFilesWithErrorRequests.yaml

```diff
@@ -0,0 +1,28 @@
+id: 191bd49e-4881-11ec-81d3-0242ac130003
+name: Tomcat - Top files with error requests 
+description: |
+  'Query shows list of files with error requests.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  TomcatEvent
+  | where TimeGenerated > ago(24h)
+  | where toint(HttpStatusCode) >= 400 and toint(HttpStatusCode) <= 599 

```

Comment:
Same on toint()

Reply:
Query works in Sentinel w/o casting toint().
Casting toint() was added to pass checks. If there is no casting toint() in files then checks fail with error